backend/infrahub/core/diff/query/empty_roots.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Generator
+
+from neo4j.graph import Node as Neo4jNode
+
+from infrahub.core.query import Query, QueryType
+from infrahub.database import InfrahubDatabase
+
+
+class EnrichedDiffEmptyRootsQuery(Query):
```

Comment:
new query to get just the roots of cached diffs without any actual diff information